source/mod_dist.f90

```diff
@@ -14,28 +14,14 @@
 !    m:          rest mass [GeV/c2]
 !    pc:         particle momentum [GeV/c]
 !    dt:         time delay with respect to the reference particle [s]
-!  - aa,zz and m are now taken into account for hisix
 !
 !  NOTA BENE:
 !  - id, gen and weight are assigned by the fluka_mod_init subroutine;
```

Comment:
I think that, for the time being, we can keep this comment and its implementation as it is.
But at a certain point, we have to merge these vars properly...